Library/Homebrew/cmd/tap.rb

```diff
@@ -63,10 +69,11 @@ def each_tap
     end
   end
 
-  def tap_args(tap_name=ARGV.named.first)
+  def tap_args()
+    tap_name, clone_target = ARGV.named[0..1]
```

Comment:
> I think that would require an extra flag

You can just check the size of `ARGV.named` inside `Homebrew#tap`. 

> I think some change to this tap_args method will be necessary

I would like to keep this as it is if possible, since this method is used in somewhere else. Also the difference between tap and any-tap only applies inside tap.rb. Let's keep the influence only inside tap.rb as well.